axis/id_matrix.f95

```diff
@@ -0,0 +1,110 @@
+module id_matrix
+
+use global_parameters
+implicit none
+
+
+contains
+
+!----------------------------------------------------------------------------------------------
+integer function get_idmatrix_2body_status(pl_id)
+! Get information about idmatrix_2body existance
+! Given:
+!   pl_id - Planet ID
+! Returns:
+!   0 - idmatrix exists and is in RAM
+!  -1 - idmatrix exists only as a file
+!  >0 - idmatrix does not exist
+    character(8):: pl_name
+    integer:: s
+    integer:: pl_id,un
+
+    if(allocated(idmatrix_2body(pl_id)%matrix)) then
+        s=0
```

Comment:
Modified this function for using "return" with no conflicts with Fortran specifications.